examples/reason-react-app/App.js

```diff
@@ -7,22 +7,16 @@ import { withPhenomicApi } from "@phenomic/preset-react-app/lib/es6/src/phenomic
 
 import * as Home from "./lib/es6/src/components/Home.bs.js";
 import * as Post from "./lib/es6/src/components/Post.bs.js";
-import ErrorPage from "./lib/es6/src/components/ErrorPage.bs.js";
+import { make as ErrorPage } from "./lib/es6/src/components/ErrorPage.bs.js";
```

Comment:
Maybe we could avoid that by doing `let default = make`?

Reply:
Yeah, that makes sense. I don't have a preference either way.